src/basics.php

```diff
@@ -120,6 +121,20 @@ function breakpoint(): ?string
     }
 }
 
+if (!function_exists('table') && (PHP_SAPI === 'cli' || PHP_SAPI === 'phpdbg')) {
+    /**
+     * REPL convenience function to load a table
+     *
+     * @param string $alias The alias name you want to get.
+     * @param array $options The options you want to build the table with.
+     * @return \Cake\ORM\Table
+     */
+    function table(string $alias, array $options = []): Table
```

Comment:
maybe `getTable()` !!??

Reply:
The idea was to be as brief as possible without going down to an unreadable ``t()``.

Reply:
I dont think in that context a get prefix makes sense.

I rather have some concerns having this now globally available, anywhere in the system, including templates.
Can be easily misused.

It would be nice if these things could be scoped into the repl only.

Reply:
`table()` is fine. For e.g. to get a collection instance the function is `collection()` not `getCollection()`.

@dereuromark I too am concerned about the misuse, but perhaps SAPI name checks can avoid general use.